BackEnd/src/main/java/com/javaschool/queue/QueueResponseServiceImp.java

```diff
@@ -0,0 +1,40 @@
+package com.javaschool.queue;
+
+import com.javaschool.entitymapper.PackageType;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+
+import java.io.IOException;
+import java.util.List;
+
+@Service
+public class QueueResponseServiceImp implements QueueResponseService{
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(QueueSenderServiceImp.class);
+
+    private QueueSenderService queueSenderService;
+    private ObjectMapper mapper;
+    TypeReference ref = new TypeReference<List<PackageType>>(){};
+
+    public QueueResponseServiceImp(final QueueSenderService queueSenderService, final ObjectMapper mapper) {
+        this.queueSenderService=queueSenderService;
+        this.mapper=mapper;
+    }
+
+    @Override
+    public List<PackageType> getType() {
```

Comment:
It's a bad practice to return null values, it can cause NullPointerExceptions like in the code below:
```
public List<String> getType() {
    return queueResponseService.getType().stream()
            .map(PackageType::getDescription).collect(Collectors.toList());
    }
```

Reply:
I have change the initialization of the variable to Collections.emptyList() instead of a null.